typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -146,6 +146,12 @@ export interface Grid3DLayerProps extends ExtendedLayerProps {
      */
     propertiesData: string | number[] | Float32Array | Uint16Array;
 
+    /**
+     * Discrete propety names to be displayed in cursor readouts.
+     * The property values are used as property name indices.
+     */
+    propertyNames?: string[];
```

Comment:
only valid for discrete properties => discretePropertyNames ? discretePropertyValueNames ? discretePropValueNames ?

Reply:
renamed to discretePropertyNames